crates/swc_ecma_transforms_module/src/common_js.rs

```diff
@@ -55,6 +58,57 @@ where
     }
 }
 
+struct LazyIdentifierVisitor {
+    scope: Rc<RefCell<Scope>>,
+    top_level_idents: AHashSet<JsWord>,
+}
+impl LazyIdentifierVisitor {
+    fn new(scope: Rc<RefCell<Scope>>) -> Self {
+        LazyIdentifierVisitor {
+            scope,
+            top_level_idents: Default::default(),
+        }
+    }
+}
+
+impl Visit for LazyIdentifierVisitor {
```

Comment:
I think this Visitor should have `visit_labelled_stmt` which ignores the identifier in label.

Reply:
Additionally, a break statement or a continue statement can have label. Those methods can be empty, just like visitor methods of import/exports.

Reply:
I also cannot make a test fail for this case, even without `visit_labeled_stmt` and `visit_continue_stmt`, here's what I tried:

```javascript
import { x } from "lib";

x:
console.log(1);

continue x;

function fn() {
    x()
}
```

This produces correctly lazy code, do you have an idea of what I'm doing wrong?

Reply:
Yeah, it's expected as currently swc does not attach context to labels, which is used by label/continue/break statements.
I was talking about being safe from future changes.